docs/source/markdown/podman-kube-apply.1.md

```diff
@@ -0,0 +1,73 @@
+-% podman-kube-apply(1)
+## NAME
+podman-kube-apply - Apply Kubernetes YAML based on containers, pods, or volumes to a Kubernetes cluster
+
+## SYNOPSIS
+**podman kube apply** [*options*] *container...* | *pod...* | *volume...*
```

Comment:
same here use wrap this in `[]` since it is optional when -f is used.

Reply:
fixed